dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -191,7 +191,7 @@ def stage_extras
       end
       # This airs on the side of not showing the warning by not showing it if the script we are in
```

Comment:
ahh, apologies for my nits here too, I didn't realize this was an existing comment!

Reply:
Oh no this was me not knowing the saying 🤦‍♀

Reply:
I very recently learned the phrase is "for all intents and purposes" instead of "for all intense purposes" 😀 Better yet, after looking it up, I learned the word [eggcorn](https://en.wikipedia.org/wiki/Eggcorn)!